tournaments/index.js

```diff
@@ -1091,6 +1091,7 @@ let commands = {
 			const name = Chat.escapeHTML(params[0].trim());
 			if (!name.length) return this.errorReply("The tournament's name cannot be blank.");
 			if (name.length > MAX_CUSTOM_NAME_LENGTH) return this.errorReply("The tournament's name cannot exceed " + MAX_CUSTOM_NAME_LENGTH + " characters.");
+			if (name.includes('|')) return this.errorReply("The tournament's name cannot include the | symbol.");
```

Comment:
Currently all the error messages for this command are using `The Tournament's name`, do you want me to change all of them to something else? Because its currently consistant with the rest.